aio_azure_clients_toolbox/clients/eventhub.py

```diff
@@ -29,7 +30,7 @@ def __init__(
         self, client: EventHubProducerClient, credential: DefaultAzureCredential
     ):
         self._client = client
-        self._credential = credential
+        self._credential = copy.deepcopy(credential)
```

Comment:
This will fail. Alternatives: shallow copy or a function to generate a credential.

Reply:
It seems like we really need to generate a new one each time: do not share between instances.